features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -3,6 +3,7 @@
   "authentication.consent.subject.prompt": true,
   "authentication.map_federated_users_to_local": false,
 
+  "session.data_source": "jdbc/WSO2IdentityDB",
```

Comment:
Having this as a default will lead to the creation of two data sources. So this may lead to the creation of two connection pools of identity DB. 
So better to keep this empty in default.json as in the code level we will be mapping the session datastore to identity datastore if this value is null. 

@ruwanta WDYT ?

Reply:
We can remove this entry from default.json and in identity.xml.j2, we can have 

```
        {% if session.data_source is defined %}
          <DataSource>
                <!-- Include a data source name (jndiConfigName) from the set of data
                    sources defined in master-datasources.xml -->
                <Name>{{session.data_source}}</Name>
            </DataSource>
            {% endif %}
```

Reply:
+1

Reply:
done